app/boards/arm/bt60/bt60_v1.zmk.yml

```diff
@@ -0,0 +1,12 @@
+file_format: "1"
+id: bt60_v1
+name: BT60 V1 Soldered
+type: board
+arch: arm
+features:
+  - keys
+  - encoder
+outputs:
+  - usb
+  - ble
+url: https://polarityworks.com
```

Comment:
This URL doesn't go anywhere currently

Reply:
Website is under development and will be up soon, we're working on it

Reply:
Do you guys have a more direct URL to the BT60? If not, could you update this in the future when you do? We can merge for now 👍

Reply:
our web dev hasn't given us the structure of the website yet unfortunately, sorry. I'll be sure to update it when they do